src/Domain/File.php

```diff
@@ -64,7 +64,7 @@ public function process(): void
                 $this->activeSniff = $sniff;
 
                 try {
-                    @$sniff->process($this, $stackPtr);
+                    $sniff->process($this, $stackPtr);
```

Comment:
By removing this, the warnings from php has now been reintroduced.

For example, try adding a file with an unclosed phpdoc comment at the end of the file.
It will result in a bunch of php warnings in the console.

Reply:
However I would really prefer if we could get around having the `@` there at all. Might be another way to silence php warnings

Reply:
Ok, I will check available ways to silence them.

Reply:
@olivernybroe I tried to reproduce the situation that you said, but I didn't see any warning! Could you provide code code? I already did this:

```
<?php

class MyClass
{
    /**
     * This is a test
}

```

And this: 
```
<?php

class MyClass
{

}
    /**
     * This is a test
```

Reply:
Might have forgotten the exact reason for this change then. It was related to errors in issue #59.

Reply:
@olivernybroe I looked into all of them and it was OK! I couldn't find a single code block which cause an error or warning. Could you test it yourself? I even tested your samples in this conversation and it was ok  => https://github.com/nunomaduro/phpinsights/pull/253/files

Reply:
Nice! 😃 
I'll test it out tomorrow then, thanks for looking into this.

Reply:
I wasn't able to reproduce this 👍 So I think it should be good.